pact/concrete-policies/collection-policy/collection-policy-v1.repl

```diff
@@ -13,12 +13,20 @@
   (marmalade-v2.abc.create-account "k:creator" (read-keyset 'creator-guard))
 (commit-tx)
 
+(begin-tx)
+(module g-utils G
```

Comment:
Does g-utils is a short-hand for guard-utils? If so I think it could be better for readability to write it in full.

Reply:
Yep, it was a shorthand for guard-utils. But I didn't want to use that name to prevent any confusion with the original guard-utils. Here it's just a simple module to have an available temporary 'always-true' guard for tests. I didn't take too much time to think to a better name. I'm OK for any suggestion better then g-utils. 

btw: looks like the team has any intention to merge this PR.
btw2: to merge this PR, I would have to resolve the conflicts with the main V2 branch. But I cant' since the tests on the main v2 branch are currently broken.